lib/index.coffee

```diff
@@ -4,10 +4,19 @@ GotoView = require('./goto-view')
 
 module.exports =
 
-  configDefaults:
-    logToConsole: false
-    moreIgnoredNames: ''
-    autoScroll: true
+  config:
+    logToConsole:
+      default: false
+      type: 'boolean'
+      description: 'Turn on logging debug information for goto commands'
+    moreIgnoredNames:
+      default: ''
+      type: 'string'
+      description: 'Whitespace-or-comma separated value of globs for filenames that goto will not generate symbols for. goto also ignores files in the core.ignoredNames setting.'
```

Comment:
For this description, I suggest: `Whitespace- or comma-separated list of globs for files that goto should skip. These files are in addition to those specified in the core.ignoredNames setting.`